Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -1792,6 +1798,8 @@ void RenderLayerCompositor::layerStyleChanged(StyleDifference diff, RenderLayer&
             layer.setNeedsPostLayoutCompositingUpdate();
             layer.setNeedsCompositingGeometryUpdate();
         }
+        if (m_renderView.settings().css3DTransformInteroperabilityEnabled() && oldStyle && recompositeChangeRequiresChildrenGeometryUpdate(*oldStyle, newStyle))
+            layer.setChildrenNeedCompositingGeometryUpdate();
```

Comment:
If the flattening layer comes and goes, this is a "configuration update", not a "geometry update".

Reply:
It appears that `setChildrenNeedCompositingConfigurationUpdate` doesn't exist, and `setChildrenNeedCompositingGeometryUpdate` implies a configuration update as well as a geometry update.